src/game_inventory.cpp

```diff
@@ -92,62 +118,222 @@ int game::inv_for_unequipped( const std::string &title )
+        }
+        // This function is for robustness. It can prevent potential crashes.
+        const islot_comestible &get_comestible( const itype *food_type ) const {
+            if( food_type == nullptr || food_type->comestible == nullptr ) {
+                static islot_comestible dummy;
+                debugmsg( "Supplied an invalid comestible." );
+                return dummy;
+            }
+            return *food_type->comestible;
+        }
+
+        const islot_comestible &get_comestible( const item &it ) const {
+            return get_comestible( get_food_type( it ) );
         }
 
-        const bool allow_buckets = location.where() == item_location::type::map;
-        return location->get_remaining_capacity_for_liquid( liquid, allow_buckets ) > 0;
-    };
+    private:
+        const player &p;
+};
 
-    return inv_map_splice( filter, title, radius,
-                           string_format( _( "You don't have a suitable container for carrying %s." ),
-                                          liquid.type_name( 1 ).c_str() ) ).get_item();
+item_location game::inv_for_comestibles( const std::string &title )
+{
+    return inv_internal( u, comestible_inventory_preset( u ),
+                         title, 1, _( "You have nothing to consume." ) );
 }
 
+class gunmod_inventory_preset: public inventory_selector_preset
+{
+    public:
+        gunmod_inventory_preset( const item &gunmod ) : gunmod( gunmod ) {
+            append_cell( [ this ]( const item & it ) {
+                std::string err;
+                it.gunmod_compatible( this->gunmod, &err );
+                return err.empty() ? string_format( "<color_ltgreen>%s</color>", _( "yes" ) ) : err;
+            }, _( "MODIFIABLE" ) );
+            // @todo Display rolls
+        }
+
+        bool is_shown( const item &it ) const override {
+            return it.is_gun() && !it.is_gunmod();
+        }
+
+        bool is_enabled( const item &it ) const override {
+            return it.gunmod_compatible( gunmod );
+        }
+
+    private:
+        const item &gunmod;
+};
+
+item_location game::inv_for_gunmod( const item &gunmod, const std::string &title )
+{
+    return inv_internal( u, gunmod_inventory_preset( gunmod ),
+                         title, -1, _( "You don't have any guns." ) );
+}
+
+class drop_inventory_preset: public inventory_selector_preset
+{
+    public:
+        drop_inventory_preset( const player &p ) : p( p ) {
+            append_cell( [ this ]( const item & it ) {
+                return to_string( it.volume() );
+            }, _( "VOLUME" ) );
+
+            append_cell( [ this ]( const item & it ) {
+                return string_format( "%.1f", convert_weight( it.weight() ) );
+            }, string_format( _( "WEIGHT(%s)" ), to_upper_case( weight_units() ).c_str() ) );
+
+            append_cell( [ this ]( const item & it ) {
+                return to_string( it.get_storage() ); //std::to_string
```

Comment:
Looks more like this calls `::to_string` from src/compatibility.h. What exactly is the comment telling (what we can't tell from the code)? In other words: what's the purpose of such a comment?